File: source/server/src/main/java/top/oldmoon/cashbook/entity/POJO/Book.java
package top.oldmoon.cashbook.entity.POJO;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Getter;
import lombok.Setter;

/**
 * 账本
 *
 * @author dingdangdog
 * @since 1.0
 */
@Setter
@Getter
@TableName("books")
public class Book {
    @TableId(type = IdType.AUTO)
    private Integer id;
    private String bookName;
    private String bookKey;
    private String createDate;
}

File: source/books/src/types/model/book.ts
export interface Book {
  id: number;
  bookName: string;
  bookKey: string;
  createDate: string;
}

export class BookQuery {
  id?: number;
  bookName?: string;
  createDate?: string;
}
/**
* 创建流水的传输实体
*/
export interface CreateBookDto {
  bookName: string;
  bookKey: string;
}
